packages/common/src/registry.ts

```diff
@@ -0,0 +1,115 @@
+import ProjectRegistryABI from "./abis/ProjectRegistry.json";
```

Comment:
if this file contains things related to the Project Registry, wouldn't `projectRegistry.ts` be a better name?